<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkGUI</class>
 <widget class="QWidget" name="WorkGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work GUI</string>
  </property>
  <widget class="QPushButton" name="workButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>WORK 5 MIN</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeOffGainLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>every hour worked = 5 upt, 3 pto, 1 vacation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="work1hourButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>WORK 1 HOUR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EndShiftButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>END SHIFT</string>
   </property>
  </widget>
  <widget class="QLabel" name="shiftLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Your shift = 12 hour shift 8:00 am - 8:00 pm</string>
   </property>
  </widget>
  <widget class="QLabel" name="AmountWorkedLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount worked today: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="UPTwarninglabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Lose UPT for each increment of 15 minutes you don't work</string>
   </property>
  </widget>
  <widget class="QLabel" name="shiftDayLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Day: Thursday</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
